### PR TITLE
[KAFKA] Add upgrade documentation

### PR DESCRIPTION
**Pull request by mbdcos**

### CODE REVIEW

**Comment by mbdcos** [comment]
retest this please

**Comment by mbdcos** [comment]
retest this please

File: frameworks/kafka/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+In order to perform auto upgrade, the existing framework must have all tasks running, no failures or errors, and all tasks must have reached their target configurations. Otherwise, auto upgrade will abort itself. Before upgrading your framework from 1.19 to 1.20, run `dcos kafka plan` from the DC/OS CLI to make sure plans/phases/steps are all completed without errors.
+
+The ZooKeeper path for task info is `dcos-service-kafka/TaskInfo`. The path for configuration is `dcos-service-kafka/Configuration`. You can roll back if auto upgrade fails; framework state will not change if there is an error in the auto upgrade process.
+
+In the new format, task names start with “kafka” (`[kafka-([0-9]+)-broker]`). By default, auto upgrade keeps a backup of old task info. Old task names start with “broker” (`[broker-[0-9]+]`).
+
+You can configure auto upgrade to delete the old task info when the upgrade process is complete. If the `CONFIG_UPGRADE_CLEANUP` environment variable is set (to any value) in Marathon, auto upgrade will delete all backups at the end. If `CONFIG_UPGRADE_DISABLE` is set, auto upgrade will be disabled.
```
Comment (sascala) [inline frameworks/kafka/release-notes.md]: Is this an env variable in the kafka service definition? that you can change from the DC/OS UI or by a JSON options file when you install Kafka from the CLI? I can't find docs on the Marathon site about these environment variables

**Comment by triclambert** [comment]
@keithchambers would you please review this before it's merged?

**Comment by mbdcos** [comment]
@keithchambers waiting +1 day.  I have to close this PR based on internal PR policy (can not keep a PR open for more than a week).

**Pull request closed without merging**